src/main/kotlin/screeps/api/Constants.kt

```diff
@@ -4,6 +4,7 @@ package screeps.api
 
 import screeps.api.structures.Structure
 import screeps.api.structures.StructureSpawn
+import kotlin.contracts.Effect
```

Comment:
I guess this isn't needed? :)

Reply:
Yeah, you're right. I'll remove it this evening